client/src/components/breadcrumbs.jsx

```diff
@@ -0,0 +1,37 @@
+//import { useState, useEffect } from 'react'
+//import PropTypes from 'prop-types'

```

Comment:
what are these? dirty hacks? 
we don't need commented code.
only comments explaining complex business logic allowed in the code.